### ISSUE TITLE
Can rewrite new script to find AUTH_TOKEN?

### ISSUE THREAD

**Issue by yuyik** [samuraitruong/coinmaster#30]
same above.

**Comment by samuraitruong** [comment]
no, this is facebook security model, we cant not scripting it, 
You can use puppeteer to login -> open the game  -> intercept the request to get token -> but this will very harmful to your facebook account because they implement a tracking bot and lock your account.
